server_addon/applications/server/tools.json

```diff
@@ -1,55 +1,149 @@
 {
     "tool_groups": [
         {
-            "environment": "{\n    \"MTOA\": \"{STUDIO_SOFTWARE}/arnold/mtoa_{MAYA_VERSION}_{MTOA_VERSION}\",\n    \"MAYA_RENDER_DESC_PATH\": \"{MTOA}\",\n    \"MAYA_MODULE_PATH\": \"{MTOA}\",\n    \"ARNOLD_PLUGIN_PATH\": \"{MTOA}/shaders\",\n    \"MTOA_EXTENSIONS_PATH\": {\n        \"darwin\": \"{MTOA}/extensions\",\n        \"linux\": \"{MTOA}/extensions\",\n        \"windows\": \"{MTOA}/extensions\"\n    },\n    \"MTOA_EXTENSIONS\": {\n        \"darwin\": \"{MTOA}/extensions\",\n        \"linux\": \"{MTOA}/extensions\",\n        \"windows\": \"{MTOA}/extensions\"\n    },\n    \"DYLD_LIBRARY_PATH\": {\n        \"darwin\": \"{MTOA}/bin\"\n    },\n    \"PATH\": {\n        \"windows\": \"{PATH};{MTOA}/bin\"\n    }\n}",
+            "name": "htoa",
+            "label": "Arnold for Houdini (example)",
+            "variants": [
+                {
+                    "name": "5-4-2-7",
+                    "label": "",
+                    "host_names": [
+                        "houdini"
+                    ],
+                    "environment": "{\n    \"HTOA_VERSION\": \"5.4.2.7\"\n}",
+                    "app_variants": []
+                }
+            ],
+            "environment": "{\n    \"_comment_\": \"{STUDIO_SW} points to software repository. Can be defined in Core addon globally\",\n\n    \"HOUDINI_PATH\": [\n        \"{STUDIO_SW}/APP/HTOA/{HTOA_VERSION}/HOUDINI{HOUDINI_VERSION}/WINDOWS/htoa-6.1.3.3_rdb15014_houdini-{HTOA_VERSION}\",\n        \"{HOUDINI_PATH}\"\n    ],\n    \"PATH\": {\n        \"windows\": [\n            \"{STUDIO_SW}/APP/HTOA/{HTOA_VERSION}/HOUDINI{HOUDINI_VERSION}/WINDOWS/htoa-6.1.3.3_rdb15014_houdini-{HTOA_VERSION}/scripts/bin\",\n            \"{PATH}\"\n        ]\n    }\n}"
```

Comment:
Is this comment a literal key/value in settings - or is it somehow used by the front end to turn this into a literal comment? I understand the thinking - I just feel it's weird that we're actually abusing the defaults for this.

I almost feel like we might be better of setting the "Placeholder text" for the settings field instead if we really just want to provide hints?

Reply:
as there is no possibility to have comments in json this way was the simplest to point out the need of setting up STUDIO_SW variable to make the example work

Reply:
also the advantage here is that you can easily remove it, while adjusting it to your studio environment. Any hardcoded hint will stay there, even though it is not valid to your setup anymore.
I would definitely not use the word `abuse` but creative use instead ;-)

Reply:
Yup - thanks. I mean, it's not a huge issue - just wondered if we could have better tools at our disposal for this. E.g. maybe a button that may apply a 'template' preset or whatever based on some built-in examples as opposed to having them predefined completely.

Anyway, nice work on cleanup!